proto-build/src/main.rs

```diff
@@ -1,338 +1,173 @@
+const COMETBFT_REPO: &str = "https://github.com/cometbft/cometbft.git";
+const IBC_GO_REPO: &str = "https://github.com/cosmos/ibc-go.git";
+const ICS23_REPO: &str = "https://github.com/cosmos/ics23.git";
+const FEEMARKET_REPO: &str = "https://github.com/skip-mev/feemarket.git";
+const SLINKY_REPO: &str = "https://github.com/skip-mev/slinky.git";
 
-macro_rules! info {
-    ($msg:expr) => {
-            println!("[info] {}", $msg)
-    };
-    ($fmt:expr, $($arg:tt)+) => {
-        info!(&format!($fmt, $($arg)+))
-    };
-}
-
-fn main() {
-    let args: Vec<String> = env::args().collect();
-    if args.len() != 2 {
-        panic!("neutron revision (commit hash) is expected as the only argument");
-    }
-    let revision = &args[1];
-
-    let tmp_build_dir: PathBuf = TMP_BUILD_DIR.parse().unwrap();
-    let proto_dir: PathBuf = PROTO_DIR.parse().unwrap();
-
-    if tmp_build_dir.exists() {
-        fs::remove_dir_all(tmp_build_dir.clone()).unwrap();
-    }
-
-    let temp_neutron_dir = tmp_build_dir.join("neutron");
+/// The Cosmos SDK commit or tag to be cloned and used to build the proto files
+const COSMOS_SDK_REV: &str = "v0.50.8-neutron";
 
-    let neutron_repo_dir: PathBuf = NEUTRON_DIR.parse().unwrap();
-    if neutron_repo_dir.exists() {
-        fs::remove_dir_all(neutron_repo_dir.clone()).unwrap();
-    }
+/// The Neutron commit or tag to be cloned and used to build the proto files
+const NEUTRON_REV: &str = "v4.1.0";
 
-    clone_neutron(revision);
-    compile_neutron_proto_and_services(&temp_neutron_dir);
-    copy_generated_files(&temp_neutron_dir, &proto_dir);
-    output_neutron_version(&proto_dir, revision);
+/// The wasmd commit or tag to be cloned and used to build the proto files
+const WASMD_REV: &str = "v0.51.0";
 
-    info!("Running rustfmt on prost/tonic-generated code");
-    run_rustfmt(&proto_dir);
+/// The cometbft commit or tag to be cloned and used to build the proto files
+const COMETBFT_REV: &str = "v0.38.10";
 
-    println!(
-        "Rebuild protos with proto-build (neutron revision: {})",
-        revision
-    );
-    fs::remove_dir_all(neutron_repo_dir.clone()).unwrap();
-}
+/// The ibc-go commit or tag to be cloned and used to build the proto files
+const IBC_GO_REV: &str = "v8.2.1";
 
-fn run_cmd(cmd: impl AsRef<OsStr>, args: impl IntoIterator<Item = impl AsRef<OsStr>>) {
-    let stdout = process::Stdio::inherit();
-    let exit_status = process::Command::new(&cmd)
-        .args(args)
-        .stdout(stdout)
-        .status()
-        .unwrap_or_else(|e| match e.kind() {
-            io::ErrorKind::NotFound => panic!(
-                "error running '{:?}': command not found. Is it installed?",
-                cmd.as_ref()
-            ),
-            _ => panic!("error running '{:?}': {:?}", cmd.as_ref(), e),
-        });
+/// The ics23 commit or tag to be cloned and used to build the proto files
+///
+/// cosmos-sdk deps for `osmo/v0.47` is `v0.9.0` but that has no buf.yml,
```

Comment:
We have Cosmos SDK 0.50, please this issue is still present. If it's not, remove comment, please

Reply:
Done